ninfo/util.py

```diff
@@ -15,7 +15,10 @@ def ishash(arg):
     return bool(_hash_re.match(arg))
 
 def get_type(arg):
-    """Return the type of the argument (mac, ip, hostname, url, hostport, or username)"""
+    """Return a list of possible types for the type of the
+    argument (mac, ip, hostname, url,hostport, and/or username)"""
+
+    potential_types = []
```

Comment:
This could just be potential_types = ["username"] since it will be added to all in any case, and remove the append at the bottom of the function.

Reply:
This was intentional.  I wanted to preserve the ordering of the types as they were previously checked to ensure that that the first type that is checked by a plugin is the same as what was checked previously.  It couldn't hurt to add a comment about this though.

What do you think?

Reply:
This one was more of a syntax thing, not super important.  Works for me